po/de.po

```diff
@@ -56,11 +57,12 @@ msgid ""
 "Add a regex, case sensitive and whole word matching option to the find dialog"
 msgstr ""
 "Es wurden die Optionen 'regexp', 'Groß- und Kleinschreibung' und 'ganze "
-"Wörter' zum Suchdialog hinzugefügt."
+"Wörter' zum Suchdialog hinzugefügt"
 
 #: data/com.github.Cogitri.gxi.appdata.xml.in:42
 msgid "Set Ctrl+W as shortcut for closing the current tab"
-msgstr "Strg+W ist jetzt das Shortcut, um den momentanen Tab zu schließen."
+msgstr ""
+"Strg+W als Verknüpfung zum Schließen der aktuellen Registerkarte einstellen"
 
```

Comment:
@SantosSi I don't think this is correct since this is in the changelog and not in the application itself.